src/board.py

```diff
@@ -224,7 +224,10 @@
     from adafruit_blinka.board.binho_nova import *
 
 elif board_id == ap_board.MICROCHIP_MCP2221:
-    from adafruit_blinka.board.microchip_mcp2221 import *
+    if "BLINKA_FORCEBOARD" not in os.environ:
```

Comment:
Can we change this to something like
```
    if "BLINKA_FORCECHIP" in os.environ and os.environ["BLINKA_FORCEBOARD"] == "MICROCHIP_MCP2221":
        from adafruit_blinka.board.fake_microchip_mcp2221 import *
    else:
        from adafruit_blinka.board.microchip_mcp2221 import *
```
Because this lies on an outside package, the possibility of os.environ["BLINKA_FORCEBOARD"] being set to something else but board_id being equal to ap_board.MICROCHIP_MCP2221 is extremely improbable, but not impossible.